charts/galoy/values.yaml

```diff
@@ -1035,20 +975,39 @@ kratos:
               default_browser_return_url: http://api:4002/login
       courier:
         smtp:
-          connection_uri: smtps://test:test@mailslurper:1025/?skip_ssl_verify=true
+          connection_uri: smtps://test:test@mailslurper:1025/?skip_ssl_verify=true # TODO: need to make this a secret and connect to an SMTP server we verified our domain with (sendgrid, mailgun etc...) see https://www.ory.sh/docs/kratos/emails-sms/sending-emails-smtp
+          from_address: noreply@blink.sv
+          from_name: Blink Verification Code
```

Comment:
@krtk6160 ,

can you make the `connection_uri` read from a secret? i am not sure how to get this value in a secret store (also i will be out on vacation next week). 

Thanks!

Reply:
It looks like if we manage the kratos secrets on our own by setting the `secret.enabled=false`, which we do in deployments, it doesn't matter what we put here - just that we put something here. The smtp uri would be available to the container thru an env var which gets the value from a [secret](https://github.com/ory/k8s/blob/master/helm/charts/kratos/templates/deployment-kratos.yaml#L174). It's a bit weird the way kratos chart is written to handle this use-case.